templates/ip6tables.j2

```diff
@@ -36,7 +36,7 @@
 ### LIMIT PORTSCAN
 ################################################################################
 -A LIMIT_PSCAN -p tcp -m tcp --syn -m recent --set --name PORT_SCAN --rsource
--A LIMIT_PSCAN -p tcp -m tcp --syn -m recent --rcheck --seconds 30 --hitcount 20 --rttl --name PORT_SCAN --rsource -j LOG --log-prefix "port scanner "
+-A LIMIT_PSCAN -p tcp -m tcp --syn -m recent --rcheck --seconds 30 --hitcount 20 --rttl --name PORT_SCAN --rsource -j LOG --log-prefix "portscanner: "
```

Comment:
This should be wrapped in `{% if iptables_portscanner_enabled %}`/`{% endif %}` so that portscan logging is actually disabled.